docs/kubernetes/windows.md

```diff
@@ -1,446 +1,358 @@
 
-#### Choosing the Windows Server version
+#### ACS-Engine
 
-If you want to deploy a specific Windows Server version, you can find available versions with `az vm image list --publisher MicrosoftWindowsServer --all -o table`
+Windows support is evolving rapidly, so be sure to use the latest ACS-Engine  version (v0.20 or later).
 
-```
-$ az vm image list --publisher MicrosoftWindowsServer --all -o table                                                                                        
+1. Browse to the ACS-Engine [releases page](https://github.com/Azure/acs-engine/releases) on GitHub.
 
-Offer                    Publisher                      Sku                                             Urn                                                                                                            Version
------------------------  -----------------------------  ----------------------------------------------  -------------------------------------------------------------------------------------------------------------  -----------------
-...
-WindowsServerSemiAnnual  MicrosoftWindowsServer         Datacenter-Core-1709-with-Containers-smalldisk  MicrosoftWindowsServer:WindowsServerSemiAnnual:Datacenter-Core-1709-with-Containers-smalldisk:1709.0.20180412  1709.0.20180412
-WindowsServerSemiAnnual  MicrosoftWindowsServer         Datacenter-Core-1803-with-Containers-smalldisk  MicrosoftWindowsServer:WindowsServerSemiAnnual:Datacenter-Core-1803-with-Containers-smalldisk:1803.0.20180504  1803.0.20180504
-```
+2. Find the latest version, and download the file ending in `-windows-amd64.zip`.
 
-You can use the Offer, Publisher and Sku to pick a specific version by adding `windowsOffer`, `windowsPublisher`, `windowsSku` and (optionally) `widndowsVersion` to the `windowsProfile` section. In this example, the latest Windows Server version 1803 image would be deployed.
+3. Extract the `acs-engine...-windows-amd64.zip` file to a working folder such as `c:\tools`
 
-```
-"windowsProfile": {
-            "adminUsername": "azureuser",
-            "adminPassword": "...",
-            "windowsPublisher": "MicrosoftWindowsServer",
-            "windowsOffer": "WindowsServerSemiAnnual",
-            "windowsSku": "Datacenter-Core-1803-with-Containers-smalldisk"
-     },
+4. Check that it runs with `.\acs-engine.exe version`
+
+```none
+PS C:\Users\patrick\acs-engine> .\acs-engine.exe version
+Version: v0.20.6
+GitCommit: 293adfda
+GitTreeState: clean
 ```
 
-## Walkthrough
+5. [optional] Add the folder you created in step 3 to your path.
 
-Once your Kubernetes cluster has been created you will have a resource group containing:
+```powershell
+$ENV:Path += ';c:\tools'
+# If you want to save the setting permanently, then run
+$oldPath = [Environment]::GetEnvironmentVariable('Path', [EnvironmentVariableTarget]::User)
+[Environment]::SetEnvironmentVariable('Path', $oldPath + ';c:\tools', [EnvironmentVariableTarget]::User)
+```

```

Comment:
Propose second optional path/alias addition for *nix/bash:

Edit $HOME/.bash_profile or $HOME/.zshrc with the following:
```
alias acs-engine="/usr/local/bin/acs-engine"
```